src/docs/self-hosted/reverse-proxy.mdx

```diff
@@ -0,0 +1,215 @@
+
+Once you have setup a reverse proxy to your Sentry instance, you should modify the `system.url-prefix` in the `config.yml` file to match your new URL and protocol. You should also update the SSL/TLS section in the `sentry/sentry.conf.py` script, otherwise you may get CSRF-related errors when performing certain actions such as configuring integrations.
+
+Keep in mind that all this setup uses single-nodes for all services, including Kafka. For larger loads, you'd need a beefy machine with lots of RAM and disk storage. To scale up even further, you are very likely to use clusters with a more complex tool, such as Kubernetes. Due to self-hosted installations' very custom nature, we do not offer any recommendations or guidance around scaling up. We do what works for us for our thousands of customers over at [sentry.io](https://sentry.io/) and would love to have you over when you feel your local install's maintenance becomes a burden instead of a joy.
+
+## Enabling HTTPS
+
+We recommend TLS termination to be done on your own dedicated load balancer or proxy. Although you can set it on the `nginx.conf` file, it is not recommended as newer self-hosted releases might alter some configurations on the file. Some examples are available on [Reverse Proxy Examples](#reverse-proxy-examples) section.
+
+## Expose Only Ingest Endpoint Publicly
+
+Certain self-hosted deployments requires the dashboard to be accessed only via internal network. But, they also need to provide public Sentry ingestion endpoint for client devices such as mobile and desktop apps. You can expose some of these endpoints publicly:
+
+- `/api/[1-9]\d*/envelope/` - Main endpoint for submitting event from SDK
+- `/api/[1-9]\d*/minidump/` - Endpoint for submitting minidump from native SDKs
+- `/api/[1-9]\d*/security/` - Endpoint for submitting security-related such as CSP errors
+- `/api/[1-9]\d*/store/` - Old endpoint for submitting event from SDK, it is deprecated.
+- `/api/[1-9]\d*/unreal/` - Endpoint for submitting crash report from Unreal Engine SDK 
+
+The `[1-9]\d+` is a regular expression string that is acquired from the project DSN.
+
+## Rate Limiting
+
+By default, Sentry does not handle rate limiting for any incoming request. On hosted version of Sentry (SaaS), it has a feature called [spike protection](https://docs.sentry.io/product/accounts/quotas/spike-protection/) which can protect you from event flood. The code for that module is not available on the public [sentry](https://github.com/getsentry/sentry) repository, it lives on the private getsentry repository instead.
+
+For self-hosted deployment, it is recommended to have a rate limiter on your dedicated load balancer to prevent such things to happen. It is highly recommended than ever if you expose your Sentry instance publicly to the internet. Some example are available on [Using Dedicated Load Balancer](#using-dedicated-load-balancer) section. 
```

Comment:
#using-dedicated-load-balancer here is now removed, we'll need to update this to get the build passing